dialogue-hauntedhouse.js

```diff
@@ -208,7 +208,7 @@ const hauntedHouseDialogue2 = [
         nextText: 14,
       },
       {
-        text: "(Lie) She wants you to find loving owners for the house, that will also adopt her cats.",
+        text: "(Lie) She wants you to find a loving owner for the house, someone who will also adopt her cats.",
```

Comment:
```suggestion
        text: "(Lie) She wants you to find a loving owner for the house who will also adopt her cats.",
```
@LaChapeliere is right, this ran over a bit. Adjusting to fit.

Reply:
I don't know how to add a suggestion to your suggestion, but there just need to be a comma between "house" and "who". Apart from that, this text works for me :)

Reply:
@LaChapeliere Updated, if that looks good to you I'll commit it in and merge this PR